content/hardware/_unlisted/kits/starter-kit/product.md

```diff
@@ -1,4 +1,5 @@
 ---
 title: Starter Kit
 certifications: [CE, UKCA]
+sku: [K000007, K010007, K020007, K030007, K040007, K050007, K060007, K090007, K110007, K120007]
```

Comment:
Missing `K130007`?

Reply:
yes but I haven't created a declaration for it yet :/ I am not sure if it is already released?

Reply:
ah I see! Ok, then we'll skip that one!